R/validations.R

```diff
@@ -262,7 +262,7 @@ validate_no_intersection <- function(x, y, msg) {
 #'
 #' ui <- fluidPage(
 #'   selectInput("arm", "Select treatment",
-#'     choices = c("ARM", "ARMCD", "ACTARM", "ACTARMCD", "TRT01P", "TRT01A"),
+#'     choices = c("ARM", "ARMCD", "ACTARM", "TRT"),
```

Comment:
Good findings.

Also, we should improve the validation message given out here:

![image](https://user-images.githubusercontent.com/8597300/229927375-eb6eb555-3da8-4040-ac6e-2c1965d3cd99.png)

The current message implies that all columns do not exist where it supposed to be just for `TRT` column.

Reply:
#821